pandas/core/indexing.py

```diff
@@ -1441,6 +1441,9 @@ def _getbool_axis(self, key, axis=None):
             axis = self.axis or 0
         labels = self.obj._get_axis(axis)
         key = check_bool_indexer(labels, key)
+        if 0 < len(labels) != len(key):
+            raise ValueError('Item wrong length %d instead of %d.' %
```

Comment:
can you use .format() for the messages itself.

I think this check should actually be in check_bool_indexer

would be really helpful to add a doc-string (and typing if you can) to that function as well